doc/src/vpicUnits.tex

```diff
@@ -37,41 +51,156 @@
 		Q &= l_{\mathrm{deck}}\left(\frac{c}{c_{\mathrm{deck}}}\right)^2\frac{\varepsilon_0}{\varepsilon_{0,\mathrm{deck}}}\frac{r_{\alpha,\mathrm{deck}}}{r_{\alpha}}.
 	\end{align}
 
-	\noindent The electric and magnetic field conversion factors $E$ and $B$ are therefore given by
+	\noindent The electric and magnetic field conversion factors $E$ and $B$ are then given by
 
 	\begin{align}
 		E &= \frac{ML}{QT^2} = \frac{1}{l_{\mathrm{deck}}}\frac{r_{\alpha,\mathrm{deck}}}{r_{\alpha}}\left(\frac{c}{c_{\mathrm{deck}}}\right)^2 \\
 		B &= E\frac{T}{L} = \frac{1}{l_{\mathrm{deck}}}\frac{r_{\alpha,\mathrm{deck}}}{r_{\alpha}}\frac{c}{c_{\mathrm{deck}}}
 	\end{align}
 
-	As an example of how this might work, one possible choice of units is $l_{\mathrm{deck}} = c/\omega_{\mathrm{pe}}$, $c_{\mathrm{deck}} = 1$, $\varepsilon_{0,\mathrm{deck}} = 1$ and $(q_e/m_e)_{\mathrm{deck}} = -1$. This gives us
+	\subsection{Particle Momenta}
 
-	\begin{align}
-		L &= \frac{c}{\omega_{\mathrm{pe}}}, \\
+	Particle momenta are the only user-facing exception to the unit system defined above. The momenta are normalised to produce a dimensionless momentum
```

Comment:
I don't see how this is an exception, you just have to know that the quantity is normalized momentum, not momentum.  It's the same as the code storing cB instead of B.

Reply:
It depends how you look at it. You can argue that p/mc is a dimensionless quantity computed from the momentum, or that it's a momentum in units of mc. I guess I am confusing things by saying it is a dimensionless momentum AND that it is an exception to the unit system.

Happy to change it to be consistent with one or the other way of looking at it, but either way it needs to be documented. I should also add something about the magnetic field normalisation/unit - I forgot about that.

Reply:
From `species_advance.h`, `float ux, uy, uz; // Particle normalized momentum`.  We need to be consistent with what's in the code.  Units of mc are different for each species, so you would have many momentum units in any given simulation.